doc/error_handling_guidelines.md

```diff
@@ -36,7 +36,7 @@ The newly designed error types are provided in [azure/cli/core/azclierror.py](ht
 |    |    | -- FileOperationError
 |    |    | -- ManualInterrupt
 |    |    | -- ... More ...
-|    |    | -- UnknownError                     # fallback of UserFault related errors
+|    |    | -- UnclassifiedUserFault            # fallback of UserFault related errors
```

Comment:
```suggestion
|    |    | -- UnclassifiedError            # fallback of UserFault related errors
```

Reply:
Others end with "Error"

Reply:
It's ok, not such a rule to be end with `Error`